src/bin/run-parser-test-suite.rs

```diff
@@ -142,7 +143,7 @@ fn print_escaped(stdout: &mut dyn Write, s: &str) {
 fn main() -> ExitCode {
     let args = env::args_os().skip(1);
     if args.len() == 0 {
-        let _ = writeln!(io::stderr(), "Usage: run-parser-test-suite <in.yaml>...");
+        _ = io::stderr().write_all("Usage: run-parser-test-suite <in.yaml>...".as_bytes());
```

Comment:
I think this is a case where [`eprintln`](https://doc.rust-lang.org/std/macro.eprintln.html#) would be better for simplicity - any perf improvement isnt worth it in this context.

Reply:
good catch, makes sense, i fixed it in a few places